pkg/collector/corechecks/servicediscovery/targetenvs/envscanner.go

```diff
@@ -0,0 +1,103 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+package targetenvs
+
+import (
+	"bytes"
+	"fmt"
+	"hash/fnv"
+	"os"
+	"strconv"
+
+	"github.com/DataDog/datadog-agent/pkg/security/ptracer"
```

Comment:
that's import caused an massive impact on the agent, and other component (https://github.com/DataDog/datadog-agent/pull/29574#issuecomment-2391880539)
why are you using this package?

Reply:
Vincent suggested to import this package to use file scanner from it, I will remove import and create local code instead - it is just 30 lines of code

Reply:
I've removed imported ptracer in recent commit